lib/widgets/content.dart

```diff
@@ -296,13 +296,16 @@ class MessageContent extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final content = this.content;
+
+    Widget child = switch (content) {
+      ZulipContent() => BlockContentList(nodes: content.nodes),
+      PollContent()  => PollWidget(messageId: message.id, poll: content.poll),
+    };
+
     return InheritedMessage(message: message,
       child: DefaultTextStyle(
         style: ContentTheme.of(context).textStylePlainParagraph,
-        child: switch (content) {
-          ZulipContent() => BlockContentList(nodes: content.nodes),
-          PollContent()  => PollWidget(messageId: message.id, poll: content.poll),
-        }));
+        child: child));
```

Comment:
This part might become clearer in the upcoming PR that will use this change. But I'm not sure that here in MessageContent is the right place to implement this behavior:

    content [nfc]: Pull out `child` variable in MessageContent.build
    
    For the edit-message feature, the Figma fades message content during
    the loading state:
      https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=4026-8563&m=dev
    This refactor prepares for that.

It feels like it might more naturally live in the parent widget.

(Which could do it just as well, right? I imagine the fading doesn't interact with the DefaultTextStyle or the InheritedMessage.)

Reply:
Indeed; yeah, I'll drop this commit.